_data/fa-owners.yml

```diff
@@ -0,0 +1,353 @@
+'API definition':
+  - area: 'API definition'
+    label: api-definition
+    owner:
+    - sq-lb-epitome
+    repos:
+    - loopback-api-definition
+    - loopback-component-explorer
+    - loopback-swagger
+
+'Boot':
+  - area: Boot
+    label: boot
+    owner:
+    - sq-lb-asteroid
+    repos:
+    - loopback-boot
+    - loopback-example-app-logic
+    smes:
+    - davidcheung
```

Comment:
FINE, whatever!  ;-)  I added the ampersand.
While doing that, I fixed a couple bugs, so that was good. 

I suggest we merge this, and continue to iterate on it.  Actually seeing the live page will make it easier to understand what needs to change.